ibis/pandas/execution/timecontext.py

```diff
@@ -0,0 +1,54 @@
+""" Implementation of compute_time_context for time context related operations
+
+Time context of a node is computed at the beginning of execution phase.
```

Comment:
Maybe to add that this is optional feature. Resulting of executing an expression without timecontext and conceptually the same as executing an expression with (-inf, inf) timecontext.

Reply:
Added this to the docstring.